### PR TITLE
Personal interaction messages to simple animals

### LINKED ISSUE #46168: Someone else opening the valve of your tank displays the wrong pronoun to you
If someone interacts with your inventory and forces you to turn on your internals, the message shown in big red text to you will say that YOU forced you to turn your internals on, not the person who is actually turning your internals on.

### PR DESCRIPTION
**Pull request by kingofkosmos**
## About The Pull Request
Adds personal messages to interactions with all simple animals by adding new attack2, friendly2, response_help2, response_disarm2 and response_harm2 -messages which contain the verb in present tense. (Also fixes my oversight of response_harm in my earlier PR, I didn't realize these were made for all intents. They should all be ok now.)

Before, the player touching the Killer Tomato with help and harm intents:
you1

After:
you2

Fixes #46168

## Why It's Good For The Game
Easier for reading comprehension.

### CODE REVIEW

**Comment by kingofkosmos** [comment]
Ready for reviewing.

**Comment by yoyobatty** [comment]
Holy shit dude, you killed it.

**Comment by kingofkosmos** [comment]
I'll add the fix to the few visible_messages into this PR, if that's ok.

**Comment by AnturK** [comment]
These variable names are horrible, make them represent what they do and add autodoc

File: code/modules/mob/living/simple_animal/guardian/types/ranged.dm [hunk 1]
```diff
@@ -131,8 +132,3 @@
 	// To stop scout mode from moving when recalled
 	incorporeal_move = FALSE
 	. = ..()
-
-/mob/living/simple_animal/hostile/guardian/ranged/AttackingTarget()
```
Comment (AnturK) [inline code/modules/mob/living/simple_animal/guardian/types/ranged.dm]: Also what's this.

**Review by AnturK — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/guardian/types/ranged.dm (hunk 1 above)
Comment (kingofkosmos) [reply code/modules/mob/living/simple_animal/guardian/types/ranged.dm]: I have no idea what happened there. I'll revert that.

**Comment by kingofkosmos** [comment]
Can I change Sergeant Araneus's variables in MetaStation.dmm like so?

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 2]
```diff
@@ -3433,7 +3433,8 @@
 	movement_type = 1;
 	name = "Sergeant Araneus";
 	real_name = "Sergeant Araneus";
-	response_help = "pets";
+	response_help_continuous = "pets";
+	response_help_simple = "pet";
```
Comment (kingofkosmos) [inline _maps/map_files/MetaStation/MetaStation.dmm]: Can I change Sergeant Araneus's variables in MetaStation.dmm like so?

Comment (AnturK) [reply _maps/map_files/MetaStation/MetaStation.dmm]: This is fine. Should get own path really but that's not in this pr scope.

**Pull request merged**